cli/domains_test.go

```diff
@@ -0,0 +1,728 @@
+// Copyright (c) Abstract Machines
+// SPDX-License-Identifier: Apache-2.0
+
+package cli_test
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strings"
+	"testing"
+
+	"github.com/absmach/magistrala/cli"
+	"github.com/absmach/magistrala/internal/testsutil"
+	"github.com/absmach/magistrala/pkg/errors"
+	svcerr "github.com/absmach/magistrala/pkg/errors/service"
+	mgsdk "github.com/absmach/magistrala/pkg/sdk/go"
+	sdkmocks "github.com/absmach/magistrala/pkg/sdk/mocks"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+)
+
+const (
+	createCommand     = "create"
+	getCommand        = "get"
+	updateCommand     = "update"
+	deleteCommand     = "delete"
+	usersCommand      = "users"
+	enableCommand     = "enable"
+	disableCommand    = "disable"
+	connectCommand    = "connect"
+	disconnectCommand = "disconnect"
+	connectionsCmd    = "connections"
+	assignCommand     = "assign"
+	unassignCommand   = "unassign"
+	shareCmd          = "share"
+	unshareCmd        = "unshare"
+)
+
+var domain = mgsdk.Domain{
+	ID:    testsutil.GenerateUUID(&testing.T{}),
+	Name:  "testdomain",
```

Comment:
```suggestion
	Name:  "testDomain",
```

Reply:
This is not a variable name but a name of a domain hence no need for it to be in camel case.

Reply:
still, I think it's more readable when in camelCase since it is a compound word